src/wpool/mongoose_wpool_redis.erl

```diff
@@ -30,7 +30,7 @@ wpool_spec(WpoolOptsIn, ConnOpts) ->
 
 
 makeargs(RedisOpts) ->
-    Host = proplists:get_value(host, RedisOpts, "localhost"),
+    Host = proplists:get_value(host, RedisOpts, "127.0.0.1"),
```

Comment:
Every backend and ci tool has different requirements for this unfortunately. Like, `mongoose_wpool_elastic` is still hardcoding `localhost` there, would be good to have them all consistent. Perhaps out of the scope of the PR though.

Reply:
actually there's problem with eredis library, it seems that they fixed it in the latest version. but they have just released it and it's buggy. hex package is broken and not building at all, GH release just crashes.